transbank/webpay/webpayplus/transaction.py

```diff
@@ -0,0 +1,161 @@
+
+        response_json = http_response.json()
+
+        try:
+            token = response_json["token"]
+            url = response_json["url"]
+        except KeyError:
+            raise Exception(response_json["error_message"])
+
+        json_data = response_json
+        transaction_create_response = TransactionCreateResponse(json_data)
+        return transaction_create_response
+
+    @classmethod
+    def commit(cls, token_ws, options=None):
+        commerce_code = WebpayPlus.commerce_code()
+        api_key = WebpayPlus.api_key()
+        base_url = WebpayPlus.integration_type_url()
+
+        if options is not None:
+            commerce_code = options.commerce_code
+            api_key = options.api_key
+            base_url = WebpayPlus.integration_type_url(options.integration_type)
+
+        headers = dict({
+            "Tbk-Api-Key-Id": commerce_code,
+            "Tbk-Api-Key-Secret": api_key,
+            "Content-Type": "application/json",
+        })
+
+        http_client = WebpayPlus.http_client
+        final_url = base_url + cls.COMMIT_TRANSACTION_ENDPOINT + "/" + token_ws
+        http_response = http_client.put(final_url, headers=headers)
+
+        response_json = http_response.json()
+
+        if response_json.get('error_message') is not None:
+            raise Exception(response_json["error_message"])
+
+        json_data = response_json
+        transaction_commit_response = TransactionCommitResponse(json_data)
+        return transaction_commit_response
+
+    @classmethod
+    def refund(cls, token, amount, options=None):
+        commerce_code = WebpayPlus.commerce_code()
+        api_key = WebpayPlus.api_key()
+        base_url = WebpayPlus.integration_type_url()
+
+        if options is not None:
+            commerce_code = options.commerce_code
+            api_key = options.api_key
+            base_url = WebpayPlus.integration_type_url(options.integration_type)
+
+        headers = dict({
+            "Tbk-Api-Key-Id": commerce_code,
+            "Tbk-Api-Key-Secret": api_key,
+            "Content-Type": "application/json",
+        })
+
+        payload = json.dumps(dict({
+            "amount": amount
+        }))
+
+        http_client = WebpayPlus.http_client
+        final_url = base_url + cls.REFUND_TRANSACTION_ENDPOINT.format(token)
+
+        http_response = http_client.post(final_url, data=payload, headers=headers)
+
+        if (http_response.status_code < 200) or (http_response.status_code > 300) or http_response is None:
+            raise Exception('Could not obtain a response from the service', -1)
+        response_json = http_response.json()
+
+        if response_json.get('error_message') is not None:
+            raise Exception(response_json["error_message"])
+
+        json_data = response_json
+        transaction_refund_response = TransactionRefundResponse(json_data)
+        return transaction_refund_response
```

Comment:
I'd prefer `return TransactionRefundResponse(json_data)` than using that variable which serves little purpose (given that the code isn't hard to read without it)

Reply:
Yes! You've got the reason! Let me do that change!